server/logger.js

```diff
@@ -0,0 +1,100 @@
+import winston from 'winston';
+import fs from 'fs';
+import { fileURLToPath } from 'url';
+import path from 'path';
+
+const { combine, timestamp, printf, colorize } = winston.format;
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const ROOT = __dirname;
+
+// Apply colors to log levels
+const colorizer = colorize({ all: true });
+
+const myFormat_console = printf(({ level, message, timestamp }) => {
+  return `${timestamp} ${level}: ${message}`;
+});
+
+const logger = new winston.createLogger({
```

Comment:
The logger creation should use `winston.createLogger` instead of `new winston.createLogger`. This is a constructor function and should not be instantiated with `new`. 
```diff
-  const logger = new winston.createLogger({
+  const logger = winston.createLogger({
```


    *This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*